### PR TITLE
Example project fixes

### PR DESCRIPTION
**Pull request by Psiiirus**
# Summary

I've just updated and configured the example project to meet the set up requirements of this library.

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅     |
| Android |    ❌     |

## Note
~~Besides everything is set upped as it should be it still doesnt work on iOS 13.1.3 with my real iPhone 7 device~~. Fixed by #97

### CODE REVIEW

**Comment by Psiiirus** [comment]
BTW i would love to discuss this on the RN Discord server if one of the maintainer has some spar time.

**Comment by Psiiirus** [comment]
Real device problem could be fixed with this solution ive also created a PR for
https://github.com/react-native-community/push-notification-ios/pull/97

File: example/ios/example.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -285,13 +288,19 @@
...
 				ORGANIZATIONNAME = Facebook;
 				TargetAttributes = {
 					00E356ED1AD99517003FC87E = {
 						CreatedOnToolsVersion = 6.2;
+						DevelopmentTeam = UL428N5782;
+						ProvisioningStyle = Automatic;
 						TestTargetID = 13B07F861A680F5B00A75B9A;
 					};
+					13B07F861A680F5B00A75B9A = {
+						DevelopmentTeam = UL428N5782;
```
Comment (Naturalclar) [inline example/ios/example.xcodeproj/project.pbxproj:301]: Could we remove the DevelopmentTeam option so anyone can use the example project? 😃

Comment (Psiiirus) [reply example/ios/example.xcodeproj/project.pbxproj:301]: Well i though you always need to use a Development Team?

Comment (Naturalclar) [reply example/ios/example.xcodeproj/project.pbxproj]: indeed, development team is required to run the example with a real device. however, other people won't be able to use your development team id

Comment (Psiiirus) [reply example/ios/example.xcodeproj/project.pbxproj]: Odd, Xcode always forces me to select one?...But this should matter know... the developer has to adjust the xcode-project anyway :)

**Pull request merged**